composer.json

```diff
@@ -6,17 +6,17 @@
   "require": {
     "php": ">=7.1",
     "spryker/cart-code-extension": "^1.0.1",
-    "spryker/cart-extension": "^1.0.0 || ^2.0.0",
+    "spryker/cart-extension": "^1.0.0 || ^2.0.0 || ^4.0.0",
     "spryker/currency": "^3.1.0",
     "spryker/discount-extension": "^1.0.0",
     "spryker/gui": "^3.7.0",
     "spryker/kernel": "^3.11.0",
     "spryker/log": "^3.0.0",
     "spryker/messenger": "^3.0.0",
     "spryker/money": "^2.0.0",
-    "spryker/product-option": "^6.0.0",
+    "spryker/product-option": "^6.0.0 || ^8.0.0",
     "spryker/propel-orm": "^1.0.0",
-    "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0",
+    "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0",
```

Comment:
Uses `HydrateOrderPluginInterface` Interface introduced in `f9ee8ff` which is before the `5.0.0` tag, appears to need  backporting and requiring the `^5.x.x` backport.

Also, there seems to be a high usage of  `\Orm\Zed\Sales\...\SpySalesDiscount` but the schema is in this `discount` module.

Reply:
Will be corrected in followup jira.

Reply:
@Incognito looks like backport will not bring any value, since usage was introduced in 6.0.0.
simply adding interface into 5.x.x will not allow this plugin to be used. Is this okey?